tests/system/python/conftest.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" Configuration system/python/conftest.py
+
+"""
+import pytest
+import subprocess
+import os
+
+__author__ = "Vaibhav Singhal"
+__copyright__ = "Copyright (c) 2018 Dianomic Systems"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+
+@pytest.fixture
+def reset_and_start_foglamp():
+    assert os.environ.get('FOGLAMP_ROOT') is not None
+    subprocess.run(["$FOGLAMP_ROOT/scripts/foglamp kill"], shell=True, check=True)
+    subprocess.run(["echo YES | $FOGLAMP_ROOT/scripts/foglamp reset"], shell=True, check=True)
+    subprocess.run(["$FOGLAMP_ROOT/scripts/foglamp start"], shell=True)
+    stat = subprocess.run(["$FOGLAMP_ROOT/scripts/foglamp status"], shell=True, stdout=subprocess.PIPE)
+    assert "FogLAMP not running." not in stat.stdout.decode("utf-8")
+
+
+def pytest_addoption(parser):
+    parser.addoption("--foglamp_url", action="store", default="localhost:8081",
```

Comment:
action="store" means?

Reply:
store the value of arg value passed in variable